contrib/gorm.io/gorm.v1/gorm_test.go

```diff
@@ -19,20 +19,23 @@ import (
 	"gopkg.in/DataDog/dd-trace-go.v1/ddtrace/tracer"
 	"gopkg.in/DataDog/dd-trace-go.v1/internal/globalconfig"
 
+	mssql "github.com/denisenkom/go-mssqldb"
 	"github.com/go-sql-driver/mysql"
 	"github.com/jackc/pgx/v4/stdlib"
 	_ "github.com/lib/pq"
 	"github.com/stretchr/testify/assert"
 	mysqlgorm "gorm.io/driver/mysql"
 	"gorm.io/driver/postgres"
+	"gorm.io/driver/sqlserver"
```

Comment:
Found the issue with the gorm dependency.

This dep pulls in a newer version of gorm.
We need to pin this to `v1.3.0` which shouldn't cause a newer gorm to be pulled.
https://github.com/go-gorm/sqlserver/blob/v1.3.0/go.mod